tests/test_commands.py

```diff
@@ -555,3 +556,60 @@ def test_info(docker_grafana, capsys, caplog):
     assert "dashboard_panels" in data["summary"]
     assert "dashboard_annotations" in data["summary"]
     assert "dashboard_templating" in data["summary"]
+
+
+def test_plugins_list(docker_grafana, capsys, caplog):
+    """
+    Verify the plugin inquiry API works.
+    """
+    # Which subcommand to test?
+    set_command("plugins list", "--format=yaml")
+
+    # Run command and capture YAML output.
+    with caplog.at_level(logging.DEBUG):
+        grafana_wtf.commands.run()
+    captured = capsys.readouterr()
+    data = yaml.safe_load(captured.out)
+
+    # Grafana 6 has 28 plugins preinstalled.
+    assert len(data) >= 28
+
+    # Proof the output is correct.
+    plugin0 = munchify(data[0])
+    assert plugin0.name.title() == "Alert List"
+    assert plugin0.type == "panel"
+    assert plugin0.id == "alertlist"
+    assert plugin0.category == ""
+    assert plugin0.enabled is True
+    assert plugin0.info.author.name in ["Grafana Project", "Grafana Labs"]
+    assert plugin0.info.version == ""
+
+
+def test_plugins_status(grafana_version, docker_grafana, capsys, caplog):
+    """
+    Verify the plugin status (metrics endpoint).
+
+    TODO: Verify a plugin which properly responds to a health check response.
+    """
+    if version.parse(grafana_version) < version.parse("8"):
+        raise pytest.skip(f"Plugin status inquiry only works on Grafana 8 and newer")
+
+    # Before conducting a plugin status test, install a non-internal one.
+    grafana = grafana_client.GrafanaApi.from_url(url=docker_grafana, timeout=15)
+    grafana.plugin.install_plugin("yesoreyeram-infinity-datasource")
+
+    # Which subcommand to test?
+    set_command("plugins status", "--format=yaml")
+
+    # Run command and capture YAML output.
+    with caplog.at_level(logging.DEBUG):
+        grafana_wtf.commands.run()
+    captured = capsys.readouterr()
+    data = yaml.safe_load(captured.out)
+
+    # Grafana 6 has 28 plugins preinstalled.
+    assert len(data) >= 28
+
+    # Proof the output is correct.
+    infinity_datasource = next(item for item in data if item["id"] == "yesoreyeram-infinity-datasource")
+    assert "go_gc_duration_seconds" in infinity_datasource["metrics"]
```

Comment:
@bhks: This is a first real integration test, using your contribution to the grafana-client library.

Reply:
Awesome work Andreas.

Reply:
Just building upon _your_ work here ;], thanks again.

Reply:
CI currently fails here, because a few more adjustments will be submitted to grafana-client to make it work.

Reply:
Also:

> TODO: Verify a plugin which properly responds to a **health check** response.

Haven't been able to discover one, yet. If that plugin sample you just shared with me, will respond to such a response, I will be happy about it.